app/routes/_gcn.user.email/email_notices.server.ts

```diff
@@ -243,11 +243,37 @@ export async function sendNewsAnnouncementEmail(
     if (newEmails) emails.push(...newEmails)
   }
 
+  const legacyEmails = await getLegacyAnnouncementReceiverEmails()
```

Comment:
- [ ] Great idea to put this in a separate function, but please also refactor the code above that gets the email addresses of real users as a function `getAnnouncementReceiverEmails`.
- [ ] Call both `getAnnouncementReceiverEmails` and `getLegacyAnnouncementReceiverEmails` concurrently using `Promise.all`.
- [ ] Most of this code should go in `email_announcements.server.ts`, not `email_notices.server.ts`.